string name
geometry_msgs/Pose pose
float32 sd_position
float32 sd_quaternion